tests/predefined/gen_predefined.go

```diff
@@ -31,7 +31,7 @@ func parseSDL(gqlSDL string) (map[string]client.CollectionDefinition, error) {
 	}
 	result := make(map[string]client.CollectionDefinition)
 	for _, col := range cols {
-		result[col.Description.Name] = col
+		result[col.Description.Name.Value()] = col
```

Comment:
> question: Why have you skipped `HasValue()` here before calling `Value()`. If IRRC the value returned is invalid unless `HasValue() == true`

question: Same as above, and other places in this file

Reply:
These collections are returned from a parsed SDL, I would be curious as to how one could declare a nameless collection in an SDL doc :)